.devcontainer/Dockerfile

```diff
@@ -140,11 +139,10 @@ ARG BUILD_PACKAGES="\
   libpq-dev \
   # https://github.com/PyMySQL/mysqlclient#linux
   default-libmysqlclient-dev \
-  pkg-config \
-  pre-commit"
+  pkg-config"
 
 # hadolint ignore=DL3042
-RUN --mount=type=cache,target=/root/.cache/pip/,id=pip-cache \
+RUN --mount=type=cache,target=/root/.cache/uv,id=pip-cache \
```

Comment:
I guess for the dev container, the idea is to save time across container launches? I've found with uv being so fast to install, manually managing the cache seems unnecessary, so you could consider dropping this bit.

Reply:
For the moment, I'll keep it, just for fewer changes.  Honestly, I'm not even sure who uses the dev container